src/views/List.jsx

```diff
@@ -1,15 +1,44 @@
+import { useState } from 'react';
+
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	const clearSearchTerm = () => {
+		setSearchTerm('');
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+			<form onSubmit={(e) => e.preventDefault()}>
```

Comment:
Thanks! The onSubmit was added to the form element because Andrew found a bug: without prevent default, when you hit enter while in the input field, it behaves like a submit button and reloads the page. This helps prevent unexpected behavior from the user perspective.

Reply:
I see, that makes total sense, though unexpected! 
I also may have been unclear above but wanted to clarify that you could probably remove the `<form>` tag entirely as implementation right now doesn't depend on it. But I do see in the issue it says explicitly to include a `<form>` element 😮 
I'm wondering if a mentor can clarify the necessity of wrapping in a form when form controls such as `onSubmit` or getting `FormData` aren't being used? Would it be purely for semantic/accessibility reasons?

Reply:
That's a really good point @xynree! I wonder also if it's better semantics to use <form> but either way good job on the implementation here with useState and onChange!

Reply:
Great feedback! While the `form` tag is not required, it's a great way to wrap related inputs in a semantic way instead of using a `div`, but it does require you to handle the `onSubmit` method as you mentioned.